cmd/acb/commands/download/download.go

```diff
@@ -63,6 +63,7 @@ var Command = cli.Command{
 		// Add all creds provided by the user in the --credential flag
 		credentials, err := graph.CreateRegistryCredentialFromList(creds)
 		if err != nil {
+			log.Println("Failed to add credentials")
```

Comment:
does the caller log the err?

Reply:
I think so. worker.go has

```
log.Debugf("Output from downloading: %s", downloadOutput)
if err != nil {
    log.Warningf("Failed to download source context. Error: %s", err)
}
```
where `downloadOutput` contains stdOut and stdErr of the command run. The added log should be included in it.

Eg.

time="2023-10-05T16:49:50Z" level=debug msg="Output from downloading: unexpected EOF\n"
time="2023-10-05T16:49:50Z" level=warning msg="Failed to download source context. Error: exit status 1"

time="2023-10-23T06:16:39Z" level=debug msg="Output from downloading: fatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\nfatal: not a git repository (or any parent up to mount point /)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n2023/10/23 06:16:39 Download complete, working directory: /workspace\n2023/10/23 06:16:39 CommitID and Branch information: {\"commitID\":\"\",\"branch\":\"\"}\n"